lib/teleterm/apiserver/handler/handler_auth.go

```diff
@@ -74,10 +100,13 @@ func (s *Handler) GetAuthSettings(ctx context.Context, req *api.GetAuthSettingsR
 	}
 
 	result := &api.AuthSettings{
-		PreferredMfa:     string(preferences.PreferredLocalMFA),
-		SecondFactor:     string(preferences.SecondFactor),
-		LocalAuthEnabled: preferences.LocalAuthEnabled,
-		AuthProviders:    []*api.AuthProvider{},
+		PreferredMfa:       string(preferences.PreferredLocalMFA),
+		SecondFactor:       string(preferences.SecondFactor),
+		LocalAuthEnabled:   preferences.LocalAuthEnabled,
+		AuthProviders:      []*api.AuthProvider{},
```

Comment:
I believe `AuthProviders` is used to list SSO options. Here Connect is more similar to the Web UI than tsh as we need to list all available options upfront so that the user can select one of them (rather than making specify one through a flag in tsh).

Reply:
Ack. From the initialization and use it looks like the field is unnecessary (always nil), but maybe it's just my view from this PR?

If it is indeed unused I would ditch it until the actually has a purpose.

Reply:
you both are right, it's not required to initialize it even though we are appending to it just below this code. 

although if we know the length of this array, should we be initializing with the `make with length` as we tend to do else where or is this trivial? but i removed this line of code

Reply:
you both are right, it's not required to initialize it even though we are appending to it just below this code.

although if we know the length of this array, should we be initializing with the make with length as we tend to do else where or is this trivial? but i removed this line of code

Reply:
you both are right, it's not required to initialize it even though we are appending to it just below this code.

although if we know the length of this array, should we be initializing with the make with length as we tend to do else where or is this trivial? but i removed this line of code

Reply:
Re: make with length, it doesn't look it matters a whole lot here, but if you decide to go for it I suggest doing it right next to the `for`, so it's easy to follow the entire logic.